prompting/datasets/random_website.py

```diff
@@ -0,0 +1,63 @@
+import random
+from typing import Optional
+from duckduckgo_search import DDGS
+import trafilatura
+from prompting.datasets.base import BaseDataset, Context, DatasetEntry
+from loguru import logger
+
+
+def load_words(file_path: str) -> list[str]:
```

Comment:
Let's use `nltk` since we already have it in our reqs, and remove `english_words.txt` file
```
from nltk.corpus import words
english_words = words.words()
```